core-bundle/src/Resources/contao/config/mimetypes.php

```diff
@@ -103,6 +103,9 @@
 	'svg'   => array('image/svg+xml', 'iconSVG.svg'),
 	'svgz'  => array('image/svg+xml', 'iconSVGZ.svg'),
 	'webp'  => array('image/webp', 'iconWEBP.svg'),
+	'avif'  => array('image/avif', 'iconAVIF.svg'),
+	'heic'  => array('image/heic', 'iconHEIC.svg'),
+	'jxl'  => array('image/jxl', 'iconJXL.svg'),
```

Comment:
@leofeyer who usually generates these icons?
The webp icon was added in https://github.com/contao-components/contao/commit/3a003eca5b9d9dfdfc10f5b0ae965d3bdddd39df?short_path=d22bb60#diff-d22bb60ec705ee366f3a5f37c2d77143cd2fc712d58b0d518ed1b7e7d2618fbd

Reply:
Done. See https://github.com/contao-components/contao/commit/44abd0b24e5508ff427b7e722ddf6f8617591fc8.